ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -207,28 +207,33 @@ class SandboxServer(actorSystemName: String, config: => SandboxConfig) extends A
 
     val stopHeartbeats = scheduleHeartbeats(timeProvider, ledger.publishHeartbeat)
 
-    val apiServer = LedgerApiServer(
-      (am: ActorMaterializer, esf: ExecutionSequencerFactory) =>
-        ApiServices
-          .create(
-            config,
-            ledgerBackend,
-            ledgerBackend,
-            ApiServices.configurationService(config),
-            SandboxServer.engine,
-            timeProvider,
-            timeServiceBackendO
-              .map(
-                TimeServiceBackend.withObserver(
-                  _,
-                  ledger.publishHeartbeat
-                ))
-          )(am, esf)
-          .withServices(List(resetService)),
-      // NOTE(JM): Re-use the same port after reset.
-      Option(sandboxState).fold(config.port)(_.apiServerState.port),
-      config.address,
-      config.tlsConfig.flatMap(_.server)
+    val apiServer = Await.result(
```

Comment:
Do we really want to wait until infinity in here?

Reply:
good point, I changed it to use `asyncTolerance`. BTW,  I am blocking here on purpose as it didn't seem reasonable to push the async behaviour further up the stack. We might do it later though.